VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -21,7 +21,7 @@
         <RelativeLayout
             android:id="@+id/announcement_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="fill_parent">
```

Comment:
I think the idea is that @bguertin is mentioning is that we shouldn't be using `layout_height=match_parent` with a vertical linear layout in general.